### PR TITLE
Preview video now using AVCaptureVideoPreviewLayer

### PR DESCRIPTION
**Pull request by tapwork**
Preview video changed from UIImageImagePickerController to AVCaptureVideoPreviewLayer

- Removed legacy code
- Updated project file
- Orientation support

### CODE REVIEW

File: Classes/NBNImageCaptureCell.m [hunk 1]
```diff
@@ -1,68 +1,102 @@
...
 @interface NBNImageCaptureCell ()
 @property (nonatomic) UIImageView *maskImageView;
-@property (nonatomic) CGSize cellSize;
+@property (nonatomic, weak) AVCaptureVideoPreviewLayer *previewCaptureLayer;
 @end
 
 @implementation NBNImageCaptureCell
 
-- (id)initWithFrame:(CGRect)frame
+#pragma mark - Init & dealloc
```
Comment (pietbrauer) [inline Classes/NBNImageCaptureCell.m]: I don't think the pragma is necessary here

File: Classes/NBNPhotoChooserViewController.m [hunk 2]
```diff
@@ -74,7 +74,10 @@ - (CGSize)setupCellSize {
         numCells++;
         cellWidth = floorf((availableWidth - (numCells - 1) * self.cellSpacing) / numCells);
     }
-    self.cellSize = CGSizeMake(cellWidth, cellWidth);
+    CGSize cellSize = CGSizeMake(cellWidth, cellWidth);
```
Comment (pietbrauer) [inline Classes/NBNPhotoChooserViewController.m]: Can't this be changed to:
```objc
self.cellSize = CGSizeMake(cellWidth, cellWidth)
return self.cellSize
```

File: Classes/NBNImageCaptureCell.m (hunk 1 above)
Comment (tapwork) [inline Classes/NBNImageCaptureCell.m]: why not ;-)

File: Classes/NBNPhotoChooserViewController.m (hunk 2 above)
Comment (tapwork) [inline Classes/NBNPhotoChooserViewController.m]: yep

File: Classes/NBNImageCaptureCell.m (hunk 1 above)
Comment (pietbrauer) [inline Classes/NBNImageCaptureCell.m]: Because it doesn't add value.

Comment (tapwork) [inline Classes/NBNImageCaptureCell.m]: I mean it's your code
but then please remove `#pragma mark - Setup SubViews` ;-)

**Comment by pietbrauer** [comment]
Released in [0.2.0](https://github.com/nerdishbynature/NBNPhotoChooser/releases/tag/0.2.0)

**Pull request merged**